README.md

```diff
@@ -567,39 +567,44 @@ Basic usage: `amclient.py <subcommand> [optional arguments]
 
 * close-completed-transfers
   * purpose: close all completed transfers (those not failed or rejected)
-  * positional argument: am_api_key - API key for Archivematica dashboard user
+  * positional argument:
```

Comment:
Is there a reason that we are not backtick-enclosing these positional argument values like we are with the optional ones? Not that you need to fix this in this PR, just asking.

We should be able to auto-generate this portion (`### Subcommands and arguments
`) of the README from the code (`amlientargs.py::SUBCOMMANDS`) itself.

Reply:
I think that's definitely something we should aim for in another Issue/PR. I just followed the style already adopted, but auto-generating this text will lead to better methods for playing with the formatting in future.